web/elm/src/Dashboard/Group.elm

```diff
@@ -121,6 +123,61 @@ view session params g =
         ]
 
 
+viewFavoritePipelines :
+    { a | userState : UserState, favoritedPipelines : Set Concourse.DatabaseID }
+    ->
+        { dragState : DragState
+        , dropState : DropState
+        , now : Maybe Time.Posix
+        , hovered : HoverState.HoverState
+        , pipelineRunningKeyframes : String
+        , pipelinesWithResourceErrors : Set ( String, String )
+        , pipelineLayers : Dict ( String, String ) (List (List Concourse.JobIdentifier))
+        , pipelineCards : List PipelineGrid.PipelineCard
+        , headers : List PipelineGrid.Header
+        , groupCardsHeight : Float
+        , pipelineJobs : Dict ( String, String ) (List Concourse.JobIdentifier)
+        , jobs : Dict ( String, String, String ) Concourse.Job
+        }
+    -> Html Message
+viewFavoritePipelines session params =
+    let
+        pipelineCardViews =
+            params.pipelineCards
+                |> List.map
+                    (\{ bounds, pipeline } ->
+                        pipelineCardView session
+                            params
+                            FavoritesSection
+                            { bounds = bounds, pipeline = pipeline }
+                            pipeline.teamName
+                            |> (\html -> ( String.fromInt pipeline.id, html ))
+                    )
+
+        headerViews =
+            params.headers
+                |> List.map
+                    (\{ bounds, header } ->
+                        headerView bounds header
+                    )
+    in
+    Html.Keyed.node "div"
+        [ id <| "dashboard-favorite-pipelines"
```

Comment:
i'm assuming (dangerous) that this id is just used for tests?

Reply:
I believe so